search-app/src/cmr/search/data/query_to_elastic.clj

```diff
@@ -47,13 +50,13 @@
   "When sorting by keyword score, set to true if we want to bin the keyword scores
   i.e. round to the nearest keyword-score-bin-size"
   {:type Boolean
-   :default false})
+   :default true})
 
 (defconfig keyword-score-bin-size
   "When sort-bin-keyword-scores is true, what should the keyword score should
   be rounded to the nearest keyword-score-bin-size"
```

Comment:
Could you rephrase this?  It doesn't sound right -  should you remove "what should"